_templates/feature/new/package.ejs.t

```diff
@@ -0,0 +1,29 @@
+---
+# helper funcs https://www.hygen.io/docs/templates/#helpers-and-inflections
+to: blocks/<%= h.inflection.dasherize(block_name) %>-block/package.json
+---
+{
+  "name": "@wpmedia/<%= h.inflection.dasherize(block_name) %>-block",
+  "version": "0.1.0",
+  "description": "<%= h.changeCase.title( block_name ) %> Block",
+  "homepage": "https://github.com/WPMedia/fusion-news-theme-blocks",
+  "license": "CC-BY-NC-ND-4.0",
+  "main": "index.js",
+  "files": [
+    "features",
+    "intl.json"
```

Comment:
No `intl.json` is generated with this - should we generate one?

Reply:
I think so. I was thinking of showing an example translation

Reply:
ok added one; updated. included your doc on lokalise as well